src/integration/java/io/pinecone/helpers/IndexManager.java

```diff
@@ -112,7 +112,7 @@ public static Pinecone createNewIndex(Pinecone pinecone, String indexName, int d
         return pinecone;
     }
 
-    public static Index createNewIndexAndConnect(Pinecone pinecone, String indexName, int dimension, IndexMetric metric, CreateIndexRequestSpec spec) throws InterruptedException, PineconeException {
+    public static Index createNewIndexAndConnectSync(Pinecone pinecone, String indexName, int dimension, IndexMetric metric, CreateIndexRequestSpec spec) throws InterruptedException, PineconeException {
```

Comment:
Is the only change here the addition of the "Sync" part of the name?

Reply:
Yup, since it returns an `Index` I just wanted to make that clearer (as opposed to `AsyncIndex`).